src/wallet/wallet.cpp

```diff
@@ -3881,11 +3918,15 @@ void CWallet::WitnessNoteCommitment(std::vector<uint256> commitments,
  * from or to us. If fUpdate is true, found transactions that already
  * exist in the wallet will be updated.
  */
-int CWallet::ScanForWalletTransactions(CBlockIndex* pindexStart, bool fUpdate)
+int CWallet::ScanForWalletTransactions(
+        CBlockIndex* pindexStart,
+        bool fUpdate,
+        bool isInitScan)
```

Comment:
In a prior version of this PR, this parameter was `performOrchardWalletUpdates`. I note that none of the tests were updated; does that mean the value chosen for the parameter in those tests happens to be identical for both semantics?